src/scss/components/ec-grid/ec-grid.scss

```diff
@@ -0,0 +1,92 @@
+$gutters: 24px;
+$number-of-columns: 12;
+
+@mixin common {
```

Comment:
Same naming problem with this one. We already had several mixins and placeholders named like this. It would override them if haven't rename it.

Reply:
This hasn't been resolved